server/websocket.go

```diff
@@ -261,6 +262,25 @@ func (p *Plugin) handleClientMsg(us *session, msg clientMessage, handlerID strin
 			"userID":      us.userID,
 			"raised_hand": ts,
 		}, &model.WebsocketBroadcast{ChannelId: us.channelID, ReliableClusterSend: true})
+	case clientMessageTypeReaction:
+		evType := wsEventUserReact
+
+		var emoji struct {
+			Name    string `json:"name"`
+			Skin    string `json:"skin"`
+			Unified string `json:"unified"`
+		}
+		if err := json.Unmarshal(msg.Data, &emoji); err != nil {
+			p.LogError(err.Error())
+		}
+
+		p.API.PublishWebSocketEvent(evType, map[string]interface{}{
+			"userID":        us.userID,
+			"emoji_name":    emoji.Name,
+			"emoji_skin":    emoji.Skin,
+			"emoji_unified": emoji.Unified,
+			"timestamp":     time.Now().UnixMilli(),
+		}, &model.WebsocketBroadcast{ChannelId: us.channelID})
```

Comment:
I think it's worth making an effort to unify client and server on a single type.

Reply:
Could you clarify what you mean?

Reply:
We are defining an anonymous struct to parse the request and return something entirely different as ws payload. I think there's an opportunity to converge on a single type (with optional fields if needed) and use the same for request and response so that we minimize places where we need to change the casing of fields.

Reply:
Ah, yes I see what you mean. I use the `toMap` pattern from one of your PRs, and now sending/receiving the ws data is a lot cleaner, thanks.